inspire_dojson/__init__.py

```diff
@@ -26,3 +26,4 @@
 
 from . import common  # noqa: F401
 from .api import marcxml2record, record2marcxml  # noqa: F401
+from .model import DoJsonError  # noqa: F401
```

Comment:
Mh, I would put in the `.errors` module instead of `.model` (but still import it here so that we can do `from inspire_dojson import DoJsonError`.)